### DISCUSSION TITLE
Building

### DISCUSSION THREAD

**Discussion by Andywmm9** [nguyenquyhy/Flight-Tracker-StreamDeck#135] [General]
I'm able to build the project fine, and run the deploy.bat file.  I see that DistributionTool.exe says the plugin was successfully exported.  Even after restarting Streamdeck, nothing shows up.  Shouldn't new controls show up on the right side?  What am I doing wrong?

**Comment by Andywmm9** [comment]
Ok, I'm not sure what DistributionTool.exe does, but it was not copying the output to %appdata%\Elgato\StreamDeck\Plugins\tech.flighttracker.streamdeck.sdPlugin.  Once I placed the output there and restarted StreamDeck, it's showing up.

**Comment by rmroc451** [comment]
Make sure you run the deploy.bat from the bin folder, not the project folder. It won't copy the right stuff otherwise

**Comment by nguyenquyhy** [comment]
If you see DistributionTool, you are probably running build.bat. That script is to create a .streamDeckPlugin file for distribution. 
If you want to just copy the build result to Stream Deck, you can run deploy.bat inside the bin folder as rmroc has mentioned above.

Probably I'll take some time to update Contribution guide for this project.
